ols/app/metrics.py

```diff
@@ -0,0 +1,11 @@
+"""Prometheus metrics that are exposed by REST API."""
+
+from prometheus_client import Counter, make_asgi_app
+
+# just an example
+llm_calls_counter = Counter("llm_calls", "LLM calls counter")
+llm_calls_failures = Counter("llm_calls_failures", "LLM calls failures")
+llm_calls_validation_errors = Counter("llm_validation_errors", "LLM validation errors")
```

Comment:
I suggest we follow naming conventions of [Prometheus metrics](https://prometheus.io/docs/practices/naming/) to name these metrics like:
- llm_calls_total
- llm_calls_failures_total
- llm_validation_errors_total
we add a "_total" for these 3 are unit-less accumulating counters.

Reply:
ok, on it. Thank you